seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/canal/CanalJsonDeserializationSchema.java

```diff
@@ -170,17 +169,18 @@ public void deserialize(ObjectNode jsonNode, Collector<SeaTunnelRow> out) throws
                 throw new IllegalStateException(format("Unknown operation type '%s'.", type));
             }
         } catch (Throwable t) {
-            if (!ignoreParseErrors) {
-                throw CommonError.jsonOperationError(FORMAT, jsonNode.toString(), t);
+            if (ignoreParseErrors) {
+                return;
             }
+            throw CommonError.jsonOperationError(FORMAT, jsonNode.toString(), t);
```

Comment:
how about revert this changed? Seem the logic did not changed.

Reply:
Yes, it's really not necessary.